docs/ci.md

```diff
@@ -24,6 +24,22 @@ page](https://github.com/instructlab/instructlab/actions) for the repository.
 These run on a variety of instance types and can be run at the discretion of
 repo maintainers.
 
+### E2E Test Coverage
+You can specify the following flags to test various features of `ilab` with the
+`basic-workflow-tests.sh` script - you can see examples of these being used within
+the E2E job configuration files found
+[here](https://github.com/instructlab/instructlab/blob/main/.github/workflows/).
+
+| Flag | Feature |
```

Comment:
maybe add a comment above the help output in the script to tell people to update this doc when options are changed?